src/components/fabSpeedDial/fabSpeedDial.js

```diff
@@ -199,7 +199,7 @@
           offsetDelay = index * delay;
 
         styles.opacity = ctrl.isOpen ? 1 : 0;
-        styles.transform = styles.webkitTransform = ctrl.isOpen ? 'scale(1)' : 'scale(0)';
+        styles.transform = styles.webkitTransform = ctrl.isOpen ? 'scale(1)' : 'scale(0.1)';
```

Comment:
It doesn't really matter as we also set opacity to 0 and set CSS `pointer-events: none`. As I noted in the description, this should be changed back when #5195 is fixed. I will add a TODO about it.